packages/liblzma/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Should we add `set -o errexit -o nounset` to make the build fail on error?

I notice a lot of `build` scripts don't do this, but some do.

Reply:
Adding this one as well.